registry/remote/example_test.go

```diff
@@ -451,6 +517,55 @@ func ExampleRepository_Fetch_artifactReferenceManifest() {
 	// {"mediaType":"application/vnd.cncf.oras.artifact.manifest.v1+json","artifactType":"example/signature","blobs":null,"subject":{"mediaType":"application/vnd.oci.image.manifest.v1+json","digest":"sha256:00e5ffa7d914b4e6aa3f1a324f37df0625ccc400be333deea5ecaa199f9eff5b","size":24}}
 }
 
+// ExampleRepository_Fetch_blobsOfArtifactManifest gives an example of pulling the blobs
+// of an artifact manifest.
+func ExampleRepository_Fetch_blobsOfArtifactManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+
+	// resolve a manifest by tag
+	tag := "formats"
+	descriptor, err := repo.Resolve(ctx, tag)
```

Comment:
We usually resolve a referrer artifact manifest by its digest.

Reply:
We may also want to call out that we get the digest from referrers API.

Reply:
I decided to fetch directly with a given descriptor.

Reply:
I don't think users would have a descriptor. 🤔